### PR TITLE
add typescript types, continued

### PR DESCRIPTION
**Pull request by ChristianMurphy**
Follow up to #7 with some additional tests

### CODE REVIEW

File: package.json
Comment (ChristianMurphy) [inline package.json]: while it's not part of the public interface.
There may be value in holding off merging this until `vfile-location` gets types as well.
Bundling that major release update in with this one.

Comment (ChristianMurphy) [reply package.json]: https://github.com/vfile/vfile-location/pull/4

Comment (wooorm) [reply package.json]: Released!

Comment (ChristianMurphy) [reply package.json]: Thanks @wooorm!
Updated this PR with the major version update

**Comment by wooorm** [comment]
Thanks!

LGTM but I‘d like a review from someone else as well before landing

**Review by Rokt33r — APPROVED** [review]
LGTM

**Review by wooorm — APPROVED** [review]

**Comment by wooorm** [comment]
Released as 3.0.0, thanks @ChristianMurphy, @Zemnmez, and @Rokt33r!

**Pull request merged**
